packages/cluster/main.tf

```diff
@@ -154,6 +154,29 @@ module "client_cluster" {
   depends_on = [google_storage_bucket_object.setup_config_objects["scripts/run-nomad.sh"], google_storage_bucket_object.setup_config_objects["scripts/run-consul.sh"]]
 }
 
+# Add Filestore instance for NFS
+resource "google_filestore_instance" "api_nfs" {
+  name     = "${var.prefix}-api-nfs"
+  tier     = "BASIC_HDD"
+  location = var.gcp_zone
+
+  file_shares {
+    name        = "api_share"
+    capacity_gb = 1024
```

Comment:
does it make sense to have 1TB disk for metadata?

Reply:
it does not, but that's the default minimum for google filestore